docs/general/installation/truenas.md

```diff
@@ -0,0 +1,534 @@
+---
+uid: installation-truenas
+title: TrueNAS SCALE
+description: Install on TrueNAS SCALE.
+sidebar_position: 4
+---
+
+- Jellyfin can be installed on iX-systems' [TrueNAS SCALE](https://www.truenas.com/truenas-scale/).
+- This documentation is written for TrueNAS SCALE v24.10.0 (Electric Eel) or higher.
+- Consider reviewing the [TrueNAS Apps documentation](https://www.truenas.com/docs/truenasapps/) if you have not previously configured applications on your system.
+
```

Comment:
This shouldn't be a list. Also, should clarify that Truenas Core is different not supported